app/grants/templates/grants/detail/tabs.html

```diff
@@ -24,17 +24,17 @@
 
     <button type="button" id="nav-transactions" href="{{grant.url}}?tab=activity" class="section-tab {% if tab == "activity" %} active {% endif %}">
       {% trans "ACTIVITY" %}
-      <span class="nav-badge">{{ activity_count }}</span>
+      <span class="nav-badge">({{ activity_count }})</span>
     </button>
 
     <button type="button" id="nav-summary" href="{{grant.url}}?tab=updates" class="section-tab {% if tab == "updates" %} active {% endif %}">
       {% trans "UPDATES" %}
-      <span class="nav-badge">0</span>
+      <span class="nav-badge">(0)</span>
```

Comment:
is this static 0 ?

Reply:
it's the default value :P it'll get overridden